packages/odf/components/attach-storage-storagesystem/attach-storage.tsx

```diff
@@ -219,6 +226,34 @@ const AttachStorage = () => {
     storageClusterLoadError ||
     pvLoadError;
 
+  const deviceSets = React.useMemo(
+    () => storageCluster?.spec?.storageDeviceSets || [],
+    [storageCluster]
+  );
```

Comment:
no need for useMemo...
```suggestion
  const deviceSets = storageCluster?.spec?.storageDeviceSets || [];
```